test_unstructured/partition/test_api.py

```diff
@@ -98,7 +100,8 @@ def test_partition_via_api_raises_with_bad_response(monkeypatch):
 def test_partition_via_api_valid_request_data_kwargs():
```

Comment:
could you remove the tests that are skipped with reason `@pytest.mark.skip(reason="Temporary skip until since API key is now required")` like this one

Reply:
done

Reply:
i think there are four those lines in total

Reply:
okay saw them thanks